README.md

```diff
@@ -1,16 +1,26 @@
-# Medusa-plugin-filestorage-local
+# Local filestorage Medusa plugin
 
-Store your images locally on your installation of medusa e-commerce.
+This repo provides the plugin to get you started serving images using your filesystem. Follow the steps below to get ready. Store your images locally on your installation of [Medusa](https://github.com/medusajs/medusa) e-commerce.
 
-## Available options (default configuration)
+## Adding plugin to Medusa.js Core
 
+1. `yarn add https://github.com/Blechlawine/medusa-plugin-filestorage-local.git`
```

Comment:
this should not be the github url, but the npm name `medusa-plugin-filestorage-local`. Also we should not just show the yarn command, but also npm and pnpm

Reply:
Done